full-stack-tests/core/src/frontend/standalone/SnapshotConnection.test.ts

```diff
@@ -4,11 +4,9 @@
 *--------------------------------------------------------------------------------------------*/
 import { assert } from "chai";
 import { Guid } from "@bentley/bentleyjs-core";
-import { ElementProps, IModel } from "@bentley/imodeljs-common";
+import { IModel } from "@bentley/imodeljs-common";
 import { IModelApp, SnapshotConnection } from "@bentley/imodeljs-frontend";
 
-// ###TODO: snapshotR1 and snapshotF1 have same fileKey (and iModelId - snapshotR2 also has same iModelId for some reason)

```

Comment:
@pmconne see here

Reply:
I see a deleted TODO above. When I navigate to that file in your diff I don't see any such line - only removal of explicit type names. Does this test currently pass or fail on your branch?

Reply:
yes, I deleted the TODO so you'd know that's not true any more. There is now always a unique key for every IModelDb. Where is your hack?

Reply:
[Here](https://github.com/imodeljs/imodeljs/blob/816a34a8faa9e44c4830ee5aea443b3ffa6ed61d/core/backend/src/IModelDb.ts#L164)

Reply:
hack removed